### PR TITLE
test: adds new unit tests to text input components

### PR DESCRIPTION
**Pull request by carlossteinzup**
### Related Issues

https://github.com/ZupIT/beagle/issues/500
https://github.com/ZupIT/beagle/issues/1081

### Description and Example

The unit tests were not testing all possible attribute types for the TextInputType.
Beagle exposis 5 attributes being:

NUMBER
DATE
EMAIL
PASSWORD
TEXT

We have created a test to guarantee the Types set on the Beagle components are the same Types rendered on the EditText view.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/beagle/src/test/java/br/com/zup/beagle/android/components/InputTextTest.kt [hunk 1]
```diff
@@ -106,4 +109,28 @@ class TextInputTest : BaseComponentTest() {
         verify(exactly = once()) { editText.isFocusable = true }
         verify(exactly = once()) { editText.isFocusableInTouchMode = true }
     }
+
+    @Test
+    fun `verify the TextInputClass on Beagle is returning the right type of TYPE_CLASS`() {
```
Comment (uziasferreirazup) [inline android/beagle/src/test/java/br/com/zup/beagle/android/components/InputTextTest.kt:114]: What do you think about following the struct of the test name? 
https://github.com/ZupIT/beagle/wiki/Tests-Rules

**Comment by uziasferreirazup** [comment]
I think there is a file don't need to be committed in this PR called`  android/gti `

File: android/beagle/src/test/java/br/com/zup/beagle/android/components/InputTextTest.kt (hunk 1 above)
Comment (carlossteinzup) [reply android/beagle/src/test/java/br/com/zup/beagle/android/components/InputTextTest.kt:114]: Done!

**Review by uziasferreirazup — APPROVED** [review]

**Review by matheusribeirozup — APPROVED** [review]

**Pull request merged**